integration_tests/tests/consistency_pardot__campaigns.sql

```diff
@@ -0,0 +1,48 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+{% set exclude_cols = [] + var('consistency_test_exclude_metrics', []) %}
```

Comment:
If we're not defining any columns by default, we can remove the [] up front. 


```suggestion
{% set exclude_cols = var('consistency_test_exclude_metrics', []) %}
```